specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-03-01/computeRPCommon.json

```diff
@@ -621,6 +621,63 @@
       },
       "description": "Specifies the hardware settings for the virtual machine."
     },
+    "ScheduledEventsPolicy": {
+      "type": "object",
+      "description": "Specifies Redeploy, Reboot and ScheduledEventsAdditionalPublishingTargets Scheduled Event related configurations.",
+      "properties": {
+        "userInitiatedRedeploy": {
+          "$ref": "#/definitions/UserInitiatedRedeploy",
+          "description": "The configuration parameters used while creating userInitiatedRedeploy scheduled event setting creation."
+        },
+        "userInitiatedReboot": {
+          "$ref": "#/definitions/UserInitiatedReboot",
+          "description": "The configuration parameters used while creating userInitiatedReboot scheduled event setting creation."
+        },
+        "scheduledEventsAdditionalPublishingTargets": {
+          "$ref": "#/definitions/ScheduledEventsAdditionalPublishingTargets",
+          "description": "The configuration parameters used while publishing scheduledEventsAdditionalPublishingTargets."
+        }
+      }
+    },
+    "UserInitiatedRedeploy": {
+      "type": "object",
+      "description": "Specifies Redeploy related Scheduled Event related configurations.",
+      "properties": {
+        "automaticallyApprove": {
+          "type": "boolean",
```

Comment:
>boolean [](http://example.com/codeflow?start=19&length=7)

you can a few Lint warnings introduced in this PR: [https://github.com/Azure/azure-rest-api-specs/pull/28257/checks?check_run_id=22643341995](https://github.com/Azure/azure-rest-api-specs/pull/28257/checks?check_run_id=22643341995)

could you please fix them ? 

for eg this one the enum way would be preferred like : 

"ApprovalMode" : "Automatic" allowes better extensibility rather than booleans.

Reply:
This schema has already been utilized by VMSS scheduledEventsPolicy and has been checked into CRP after appropriate discussions.